android/java/org/chromium/chrome/browser/crypto_wallet/activities/AssetDetailActivity.java

```diff
@@ -146,21 +134,55 @@ public void onClick(View v) {
             }
         });
         Button btnSend = findViewById(R.id.btn_send);
-        btnSend.setOnClickListener(new View.OnClickListener() {
-            @Override
-            public void onClick(View v) {
-                Utils.openBuySendSwapActivity(AssetDetailActivity.this,
-                        BuySendSwapActivity.ActivityType.SEND, mAssetSymbol);
-            }
-        });
+        btnSend.setOnClickListener(v
+                -> Utils.openBuySendSwapActivity(AssetDetailActivity.this,
+                        BuySendSwapActivity.ActivityType.SEND, mAssetSymbol));
+
         mBtnSwap = findViewById(R.id.btn_swap);
-        mBtnSwap.setOnClickListener(new View.OnClickListener() {
-            @Override
-            public void onClick(View v) {
-                Utils.openBuySendSwapActivity(AssetDetailActivity.this,
-                        BuySendSwapActivity.ActivityType.SWAP, mAssetSymbol);
-            }
-        });
+
+        if (AssetUtils.isAuroraAddress(mContractAddress, mAssetSymbol)) {
+            mBtnSwap.setVisibility(View.GONE);
+            mBtnBridgeToAurora = findViewById(R.id.btn_aurora_bridge);
+            mBtnBridgeToAurora.setVisibility(View.VISIBLE);
+            Spanned auroraLearnMore = Utils.createSpanForSurroundedPhrase(
+                    this, R.string.brave_wallet_aurora_modal_description, (v) -> {
+                        TabUtils.openUrlInNewTab(false, WalletConstants.URL_BRIDGE_RISKS);
+                        TabUtils.bringChromeTabbedActivityToTheTop(this);
+                    });
+
```

Comment:
Could we use `openLinkWithFocus` here as well?

Reply:
yes and done https://github.com/brave/brave-core/commit/03f5d4be628edaed759365bd77a2ab5f5e817e97